engine/game/private/game_module.cpp

```diff
@@ -100,6 +103,9 @@ ModuleTickOrder GameModule::Init(Engine& engine)
 
     applicationModule.GetActionManager().SetGameActions(GAME_ACTIONS);
 
+    // Capture the static shadows
+    rendererModule.GetRenderer()->GetShadowPipeline().RequestStaticShadowUpdate();
```

Comment:
why is the game responsible of getting the shadows ready ?? this would be something to make sure happens